templates/create.tmpl.php

```diff
@@ -24,5 +24,5 @@
 
 ?>
 
-<div id="create-poll"></div>
+<div id="app-polls"></div>
```

Comment:
I guess you can get rid of that template file :wink:

Reply:
> I guess you can get rid of that template file 😉

Yep.